src/h2_stream.erl

```diff
@@ -114,10 +114,10 @@ start_link(StreamId, Connection, CallbackModule, Socket) ->
 send_pp(Pid, Headers) ->
     gen_fsm:send_event(Pid, {send_pp, Headers}).
 
--spec send_frame(pid(), http2_frame:frame()) ->
+-spec send_data(pid(), http2_frame:frame()) ->
```

Comment:
there isn't actually a data frame type. we could do `{frame_header(), h2_frame_data:payload()}` but I'm not sure it's worth it